### PR TITLE
cumulus-3562 fix reported vulnerabilities

### PR DESCRIPTION
**Pull request by etcart**
fixes critical vulnerability and whitelists a high vuln that's being flagged

## Changes

* updated crypto-js and aws package that depends on it to escape vulnerable version
* whitelist lodash.pick

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [x] Integration tests

### CODE REVIEW

File: packages/aws-client/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@aws-sdk/client-api-gateway": "^3.447.0",
+    "@aws-sdk/client-api-gateway": "^3.499.0",
```
Comment (npauzenga) [inline packages/aws-client/package.json:47]: Was this a vulnerability?

File: CHANGELOG.md [hunk 2]
```diff
@@ -51,6 +51,8 @@ terraform values and it will also update the database cluster to use the new con
   - Updated Kinesis code to AWS SDK v3
 
 ### Fixed
+- **CUMULUS-3562**
+  - Removed unnecessary dev dependency jsdoc-to-html
```
Comment (npauzenga) [inline CHANGELOG.md:55]: This doesn't seem to match the changes?

**Review by npauzenga — CHANGES_REQUESTED** [review]
Just a couple questions

File: packages/aws-client/package.json (hunk 1 above)
Comment (etcart) [reply packages/aws-client/package.json:47]: 3.447 imported crypto-js at a lower version as a dependency (whichis the actual vulnerability)
. 3.499 doens't

File: CHANGELOG.md (hunk 2 above)
Comment (etcart) [reply CHANGELOG.md:55]: ah! thank you!

File: packages/aws-client/package.json (hunk 1 above)
Comment (npauzenga) [reply packages/aws-client/package.json:47]: Ah ok, got it.

**Review by npauzenga — APPROVED** [review]
Looks good to me!

**Pull request merged**